src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/json/ReadJSONStepExecution.java

```diff
@@ -79,6 +88,52 @@ protected JSON doRun() throws Exception {
             json = JSONSerializer.toJSON(step.getText().trim());
         }
 
+        if (ObjectUtils.defaultIfNull(step.getReturnPojo(), Boolean.getBoolean(ReadJSONStep.RETURN_POJO_PROPERTY_NAME))) {
```

Comment:
```suggestion
        if (ObjectUtils.defaultIfNull(step.getReturnPojo(), ReadJSONStep.RETURN_POJO_DEFAULT)) {
```

Reply:
Change done in latest rebase opun master.